packages/pg-meta/test/db/00-init.sql

```diff
@@ -1,4 +1,5 @@
-
+-- We apply our test seedings to template1 so every new created db will have the same structure
+\c template1
```

Comment:
This is a little tricks to avoid the need to "seed" each new "created" database at jest/typescript level. That way each newly created database for each test will have our tables initialized.